From 0540dc668cbdd08a8c9c392aa0ab651312670656 Mon Sep 17 00:00:00 2001
From: Konstantin Tokarev <annulen@yandex.ru>
Date: Fri, 18 Sep 2020 00:20:43 +0300
Subject: [PATCH 1/1] Change libxml2 ICU dependency

---
 recipes/libxml2/all/conanfile.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes/libxml2/all/conanfile.py b/recipes/libxml2/all/conanfile.py
index 6d944a1c..843cfd3e 100644
--- a/recipes/libxml2/all/conanfile.py
+++ b/recipes/libxml2/all/conanfile.py
@@ -64,7 +64,7 @@ class Libxml2Conan(ConanFile):
         if self.options.iconv:
             self.requires("libiconv/1.16")
         if self.options.icu:
-            self.requires("icu/64.2")
+            self.requires("icu/67.1@qtproject/stable")
 
     def build_requirements(self):
         if self.settings.compiler != "Visual Studio" and tools.os_info.is_windows and os.environ.get("CONAN_BASH_PATH", None) is None:
-- 
2.28.0

